index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" <link rel="stylesheet"
+        href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <div class="wraper">
+        <header>
+            <p>My Logo</p>
+            <nav><i class="fa-solid fa-bars"></i></nav>
```

Comment:
- [ ] Additionally, we have to try as much as possible to stick to [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). This includes the padding, margin, font, color, etc. Therefore, I suggest that you ensure the hamburger is vertically centered in alignment with the logo and also its color matches the one of Figma☺️ .

Your design 😎 
![Screenshot 2023-02-21 132800](https://user-images.githubusercontent.com/99394374/220319536-ef97d164-9f77-4023-8ac5-d5a9690f0470.png)